packages/ui/src/composites/wui-input-cancel/index.ts

```diff
@@ -0,0 +1,29 @@
+import { html, LitElement } from 'lit'
```

Comment:
Let's make this component more reusable i.e. something like general `wui-input-element` where any icon can be provided.